code/__DEFINES/flags.dm

```diff
@@ -1,162 +1,162 @@
-	if(resistance_flags & FREEZE_PROOF)
-		to_chat(user, "[src] is made of cold-resistant materials.")
-	if(resistance_flags & FIRE_PROOF)
-		to_chat(user, "[src] is made of fire-retardant materials.")
-		return TRUE
-
-/obj/item/clothing/clothing_resistance_flag_examine_message(mob/user)
-	. = ..()
-	if(.)
-		return
-	if(max_heat_protection_temperature == FIRE_IMMUNITY_MAX_TEMP_PROTECT)
-		to_chat(user, "[src] is made of fire-retardant materials.")
-		return TRUE
-
-/obj/item/clothing/head/clothing_resistance_flag_examine_message(mob/user)
-	. = ..()
-	if(.)
-		return
-	if(max_heat_protection_temperature == (HELMET_MAX_TEMP_PROTECT || SPACE_HELM_MAX_TEMP_PROTECT || FIRE_HELM_MAX_TEMP_PROTECT))
-		to_chat(user, "[src] is made of thermally insulated materials and offers some protection to fire.")
-		return TRUE
-
-/obj/item/clothing/gloves/clothing_resistance_flag_examine_message(mob/user)
-	. = ..()
-	if(.)
-		return
-	if(max_heat_protection_temperature == GLOVES_MAX_TEMP_PROTECT)
-		to_chat(user, "[src] is made of thermally insulated materials and offers some protection to fire.")
-		return TRUE
-
-/obj/item/clothing/shoes/clothing_resistance_flag_examine_message(mob/user)
-	. = ..()
-	if(.)
-		return
-	if(max_heat_protection_temperature == SHOES_MAX_TEMP_PROTECT)
-		to_chat(user, "[src] is made of thermally insulated materials and offers some protection to fire.")
-		return TRUE
-
-/obj/item/clothing/suit/clothing_resistance_flag_examine_message(mob/user)
-	. = ..()
-	if(.)
-		return
-	if(max_heat_protection_temperature == SPACE_SUIT_MAX_TEMP_PROTECT)
-		to_chat(user, "[src] is made of thermally insulated materials and offers some protection to fire.")
-		return TRUE
-
-//tesla_zap
-#define TESLA_MACHINE_EXPLOSIVE		(1<<0)
-#define TESLA_ALLOW_DUPLICATES		(1<<1)
-#define TESLA_OBJ_DAMAGE			(1<<2)
-#define TESLA_MOB_DAMAGE			(1<<3)
-#define TESLA_MOB_STUN				(1<<4)
-
-#define TESLA_DEFAULT_FLAGS ALL
-#define TESLA_FUSION_FLAGS TESLA_OBJ_DAMAGE | TESLA_MOB_DAMAGE | TESLA_MOB_STUN
-
-//EMP protection
-#define EMP_PROTECT_SELF (1<<0)
-#define EMP_PROTECT_CONTENTS (1<<1)
-#define EMP_PROTECT_WIRES (1<<2)
-
-//Mob mobility var flags
-#define MOBILITY_MOVE			(1<<0)		//can move
-#define MOBILITY_STAND			(1<<1)		//can, and is, standing up
-#define MOBILITY_PICKUP			(1<<2)		//can pickup items
-#define MOBILITY_USE			(1<<3)		//can hold and use items
-#define MOBILITY_UI				(1<<4)		//can use interfaces like machinery
-#define MOBILITY_STORAGE		(1<<5)		//can use storage item
-#define MOBILITY_PULL			(1<<6)		//can pull things
-
-#define MOBILITY_FLAGS_DEFAULT (MOBILITY_MOVE | MOBILITY_STAND | MOBILITY_PICKUP | MOBILITY_USE | MOBILITY_UI | MOBILITY_STORAGE | MOBILITY_PULL)
-#define MOBILITY_FLAGS_INTERACTION (MOBILITY_USE | MOBILITY_PICKUP | MOBILITY_UI | MOBILITY_STORAGE)
-
-// radiation
-#define RAD_PROTECT_CONTENTS (1<<0)
-#define RAD_NO_CONTAMINATE (1<<1)
-
-//alternate appearance flags
-#define AA_TARGET_SEE_APPEARANCE (1<<0)
+/*

```

Comment:
Revert the line ending changes

Reply:
how would I do that? I've reset the file to master and nothing changed

Reply:
https://stackoverflow.com/a/11341700/3976486